tutorials/assetlib/uploading_to_assetlib.rst

```diff
@@ -63,6 +63,10 @@ library a better place for all users.
   if other people wish to contribute to your asset. See: the
   :ref:`doc_gdscript_styleguide` or the :ref:`doc_c_sharp_styleguide`.
 
+* If you have screenshots in your repo, place them in their own subfolder
+  and add a **.gdignore** file in the same folder (note: **gd**, not **git**).
+  This prevents Godot from importing your screenshots.
```

Comment:
This is already mentioned in Project Organization, but it doesn't hurt to do it here as well:

```suggestion
* If you have screenshots in your repo, place them in their own subfolder
  and add a **.gdignore** file in the same folder (note: **gd**, not **git**).
  This prevents Godot from importing your screenshots.
  On Windows, open a command prompt in the project window and use ``type nul > .gdignore``
  to create a file whose name starts with a period.
```